components/apimgt/org.wso2.carbon.apimgt.keymgt/pom.xml

```diff
@@ -47,6 +47,11 @@
 			<groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
 			<artifactId>org.wso2.carbon.identity.oauth</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.wso2.carbon.identity.governance</groupId>
+			<artifactId>org.wso2.carbon.identity.recovery</artifactId>
+			<version>1.0.2</version>
```

Comment:
Shall we move version to the parent pom?

Reply:
resolved